src/AnthroLink-192019-03-31-19.log

```diff
@@ -0,0 +1,2 @@
+{"message":"Hello World!~ in log.js","level":"info"}
```

Comment:
logs are not to be commited. As mentioned it should be in  "logs" folder and to be added in .gitignore as already mentioned.